images/stretch/hamlet/Dockerfile

```diff
@@ -41,7 +41,10 @@ RUN chmod -R u+rwx /build/scripts && \
 
 RUN echo "Cloning images on $(date +%s)"
 RUN /build/scripts/build_hamlet.sh && \
-    /build/scripts/clone.sh
+    /build/scripts/clone.sh \
```

Comment:
Should both of these now be in the base tag? 
Then hamlet is ready to go in the base and anything above that is add-ons to support a user case

Reply:
Yep, I questioned this myself. Happy to move it now.